### ISSUE TITLE
[a11y] Issue 37: Unclear reading order in "Writing Comments" section

### ISSUE THREAD

**Issue by kk-slider18** [NewJerseyStyle/polis#133]
**Description:**
"The reading order of the ""Writing Comments"" section is difficult to follow. A full review of reading order and focus behavior is needed.

eg. 
1. The page lacks a proper \<h2> heading for ""Tips for Writing Statements,"" yet the screen reader reads it in an unexpected order.
2. When focusing on the text field, the screen reader first reads a long list of previously unread content before announcing the label and input field type."

**Component/Area:** Open Participation Page

**Source:** Screen Reader

**Affected Elements:** "Writing Comments" section

**Comment by Natasha1110** [comment]
This may be caused by DOM order. Screen readers follow the structure of your HTML (DOM), to determine the reading order. So, we may start there and make sure that these elements are in the visual order being seen on screen.

**Comment by YumiChen** [comment]
Was checking this page: https://civictech-polis.azurewebsites.net/8rspm4mzbb?ui_lang=en

1. The only h2 tag on this page is currently invisible, but it would be read by the screen reader:
https://github.com/CivicTechTO/polis/blob/94e1362f33eaf4765afb6716bb0d8af900361e8c/client-participation/js/templates/comment-form.handlebars#L56

Probably we can see if we want to have other texts in this page as `h2`/ headings of sections

2.
The texts that are currently being read when the text field is focused:
![Image](https://github.com/user-attachments/assets/a13abdcf-1278-4872-b650-90b569c68ac9)

The texts read are not visible in the page, but still would be read by the screen reader:
https://github.com/CivicTechTO/polis/blob/94e1362f33eaf4765afb6716bb0d8af900361e8c/client-participation/js/templates/comment-form.handlebars#L55-L62

Probably we could see if we want to merge these texts into visible contents above the text field
